meilisearch-lib/src/dump/compat/v3.rs

```diff
@@ -134,7 +134,7 @@ impl Processed {
     fn update_task(self, task: &mut Task) {
         self.from.update_task(task);
 
-        let event = TaskEvent::Succeded {
```

Comment:
This is an enum variant that we want to keep that way as it must be compatible with the previous versions that have this typo.